### DISCUSSION TITLE
npm run dev error

### DISCUSSION THREAD

**Discussion by klunkmillan** [wprig/wprig#772] [Q&A] [answered]
I pulled down a site I'm working on with wprig, ran npm run dev is vscode, and I keep getting an error preventing it from spinning up:
10:12:31] 'php' errored after 1.01 s
[10:12:31] Error in plugin "gulp-phpcs"
Message:
    Cannot find "/Users/jerry/Local Sites/nissan-alliance-innovation-lab/app/public/wp-content/themes/wprig-master/vendor/bin/phpcs"
<img width="431" alt="Screen Shot 2021-12-09 at 10 14 36 AM" src="https://user-images.githubusercontent.com/89919784/145424016-83c026ae-0f9b-4a57-a81a-8d305968543f.png">
Anone no how I can resolve this?

**Comment by LukaszJaro** [comment, marked as answer]
Try suggestions in this thread https://github.com/wprig/wprig/issues/637 or here https://stackoverflow.com/questions/62168730/wp-rig-error-running-npm-run-build-cannot-find-vendor-bin-phpcs
